test/providers/provider_level_zero.cpp

```diff
@@ -5,9 +5,12 @@
 #include <dlfcn.h>
 #include <level_zero/ze_api.h>
 
+#include "ipcFixtures.hpp"
 #include "pool.hpp"
 #include "umf/providers/provider_level_zero.h"
 
+#include <mutex>
```

Comment:
is it needed...?

Reply:
The `InitLevelZero` function uses `std::call_once` which requires this header.